src/graph/graph_apis.cc

```diff
@@ -96,6 +96,45 @@ DGL_REGISTER_GLOBAL("graph_index._CAPI_DGLGraphCreate")
     *rv = ghandle;
   });
 
+DGL_REGISTER_GLOBAL("graph_index._CAPI_DGLGraphCSRCreate")
```

Comment:
Similar to the rationale of `from_csr_matrix`, I feel that a utility function that creates a shared memory object from CSR + `DGLGraphCSRCreateMMap` would be cleaner.

Reply:
this is to create a CSR graph. If `shared_mem_name` is provided, it creates shared memory. otherwise, allocates regular memory.

Reply:
Got it